velox/dwio/common/tests/ReadFileInputStreamTests.cpp

```diff
@@ -73,3 +75,36 @@ TEST(ReadFileInputStream, VReadIOBufs) {
   std::vector<std::string> expected = {"aaaaab", "bcccc"};
   EXPECT_EQ(result, expected);
 }
+
+TEST(ReadFileInputStream, LocalReadFile) {
```

Comment:
Consider to add test class for ReadFileInputStreamTest with TEST_F and then handle local file system registration in setup call?

Reply:
Done.